orc8r/cloud/docker/controller/apidocs/v1/swagger.yml

```diff
@@ -3100,8 +3100,7 @@ paths:
       - $ref: '#/parameters/network_id'
       responses:
         "200":
-          description: List of all MSISDNS in the network, mapped to their subscriber
-            ID
+          description: List of all MSISDNS in the network, mapped to their subscriber ID
```

Comment:
let's try to keep one PR per feature in the future. It's not that I don't like to review your code haha, but it's easier to rollback as we squash before merging to master.

Reply:
This file is generated. The change looks like it's coming from the go.mod changes, which in turn come from the changed tool dependencies

So it's a necessary part of this PR

Reply:
oh my bad!